### PR TITLE
Mk radiotherapy coded volume

### PR DESCRIPTION
**Pull request by markkramerus**
So lots of changes here to radiotherapy, and also a bunch to get a clean build.

### CODE REVIEW

**Review by masnick — APPROVED** [review]

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ In the event there are differences between the page renderings in this IG and th
...
 
 This IG was authored by the MITRE Corporation using [FHIR Shorthand (FSH)](http://hl7.org/fhir/uv/shorthand/) and [SUSHI](https://fshschool.org), a free, open source tool-chain from [MITRE Corporation](https://www.mitre.org/).
```
Comment (masnick) [inline input/pagecontent/index.md:82]: Should either be `tool chain` or `toolchain`:

```suggestion
This IG was authored by the MITRE Corporation using [FHIR Shorthand (FSH)](http://hl7.org/fhir/uv/shorthand/) and [SUSHI](https://fshschool.org), a free, open source toolchain from [MITRE Corporation](https://www.mitre.org/).
```

File: input/pagecontent/terminology.md [hunk 2]
```diff
@@ -45,97 +46,37 @@ The following table presents the external code systems (and naming conventions)
...
+The "catch code" code system was established to represent "Other, specify: ___________" as often offered as the last in a list of choices. In addition, this approach prevents certain FHIR validation problems that would result from use of extensible bindings. A detailed explanation is found [here][CatchCodeCS].
+
+#### Resource Identifier Codes
+
+Certain resources, particularly in the radiotherapy area, are missing appropriate LOINC codes. Local codes have been defined to fill this gap, at least temporarily.
+
+**Links:**
+
+* [All code systems defined in mCODE](http://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/master/artifacts.html#terminology-code-systems)
+* [All value sets defined in mCODE](http://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/master/artifacts.html#terminology-value-sets)
```
Comment (masnick) [inline input/pagecontent/terminology.md:80]:
```suggestion
* [All code systems defined in mCODE](artifacts.html#terminology-code-systems)
* [All value sets defined in mCODE](artifacts.html#terminology-value-sets)
```

**Review by masnick — COMMENTED** [review]
Please see comments below.

**Comment by masnick** [comment]
@markkramerus FYI there's no intro Markdown for https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/mk-radiotherapy-coded-volume/StructureDefinition-mcode-radiotherapy-anatomic-volume.html

File: input/fsh/SD_Radiotherapy.fsh [hunk 3]
```diff
@@ -44,26 +38,26 @@ RuleSet: RadiotherapyPhaseCommon
...
     RadiotherapyFractionsDelivered named fractionsDelivered 0..1 MS and
-    RadiotherapyDoseDelivered named doseDelivered 0..* MS
-* extension[doseDelivered].extension[fractionsDelivered] 0..0
-* extension[doseDelivered].extension[fractionsDelivered] ^short = "Not used in this profile."
-* extension[doseDelivered].extension[fractionsDelivered] ^definition = "Record the fractions delivered in this phase in the top-level extension by the same name."
-* extension[doseDelivered].extension[fractionsDelivered] ^definition = " "
+    RadiotherapyDoseDeliveredToAnatomicVolume named doseDeliveredToAnatomicVolume 0..* MS
+* extension[doseDeliveredToAnatomicVolume].extension[fractionsDelivered] 0..0
+* extension[doseDeliveredToAnatomicVolume].extension[fractionsDelivered] ^short = "Not used in this profile."
+* extension[doseDeliveredToAnatomicVolume].extension[fractionsDelivered] ^definition = "Record the fractions delivered in this phase in the top-level extension also named fractionDelivered."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/SD_Radiotherapy.fsh:44]: It's complicated, but the RT SMEs put the fractions up one level in this profile, as the definition indicates. The extension is there in the first place to make the RadiotherapyDoseDeliveredToAnatomicVolume reusable.

File: input/fsh/SD_Radiotherapy.fsh [hunk 4]
```diff
@@ -44,26 +38,26 @@ RuleSet: RadiotherapyPhaseCommon
...
 * extension[fractionsDelivered] ^short = "Number of Fractions Delivered"
 * extension[fractionsDelivered] ^definition = "The number of fractions delivered during this phase."
-* bodySite ^definition = "Not used in this profile. A more detailed description of the treatment volume should be entered in the RadiotherapyDoseDelivered.volumeDescription."
+* bodySite ^short = "Not used in this profile."
+* bodySite ^definition = "Not used in this profile. Each anatomic volume must be represented by a BodyStructure resource that conforms to the RadiotherapyAnatomicVolume profile, and referenced in the extension RadiotherapyDoseDeliveredToAnatomicVolume.anatomicVolume."
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/SD_Radiotherapy.fsh:48]: He's got a point.

File: input/fsh/SD_Radiotherapy.fsh [hunk 5]
```diff
@@ -118,36 +112,51 @@ Description: "The number of sessions in a course of radiotherapy."
...
+* description ^definition = "A text description of the anatomic volume, containing any additional information above and beyond the location and locationQualifier that describe the anatomic volume."
+* morphology from RadiotherapyVolumeTypeVS (extensible)
+* location from RadiotherapyTreatmentLocationVS (required)
+* location ^short = "Anatomical location code."
+* location ^definition = "A code specifying the body structure comprising the anatomic volume. The codes do not include laterality, which if applicable MUST be specified in the locationQualifier."
+* locationQualifier from RadiotherapyTreatmentLocationQualifierVS (extensible)
+* identifier and location and locationQualifier and description and patient and morphology MS
 
 
 Invariant:  mcode-volume-description-or-id-required
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/fsh/SD_Radiotherapy.fsh:157]: No objection to changing it to "Invariant:  mcode-description-or-id-required"

File: input/pagecontent/conformance-general.md [hunk 6]
```diff
@@ -58,7 +58,7 @@ Note that the requests below may return resources associated with patients who a
     * [TumorMarkerTest]\: `GET [base]/Observation?code:in=http://hl7.org/fhir/us/mcode/ValueSet/mcode-tumor-marker-test-vs`
 
 * Assessment
-  * [ComorbiditiesElixhauser]\: `GET [base]/Observation?code=mcode-comorbidity`
+  * [ComorbiditiesElixhauser]\: `GET [base]/Observation?code=http://loinc.org|78923-0`
```
[the remark this answers is not in the document]
Comment (markkramerus) [reply input/pagecontent/conformance-general.md:61]: I agree, but we also say "Observation resources associated with an [mCODE patient] with `Observation.code` of LNC#78923-0 MUST conform to this profile."

File: input/pagecontent/index.md (hunk 1 above)
Comment (markkramerus) [reply input/pagecontent/index.md:82]: 2.5M hits on "toolchain" versus 1.1M hits on "tool chain" so toolchain it is.

**Comment by markkramerus** [comment]
> @markkramerus FYI there's no intro Markdown for https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/mk-radiotherapy-coded-volume/StructureDefinition-mcode-radiotherapy-anatomic-volume.html

I added one.

**Pull request merged**